.adr/0010-interest-bearing-channel.md

```diff
@@ -0,0 +1,128 @@
+# 0010 - Interest Bearing Channel Application
+
+# Status
+
+Accepted
+
+# Context
+
+For a permissionless state channel network (SCN) to be sustainable, incentive structures must exist for each class of system actor (service providers, service consumers, & liquidity providers).
+
+Our work generally presumes the existence of incentives that motivate service providers & consumers. The design work for us is in mechanisms to incentivize intermediaries (liquidity providers) into the network.
+
+# Prior Art
+
+The Bitcoin Lightning Network (LN) is an in-production payment channel network (PCN) with a fee mechanism.
+
+## Similarities to Nitro-SCN
+
+In a PCN, payments between A and B are routed through intermediaries I*1, ..., I_n, where balances are pairwise adjusted along the route. Capital in intermediary channels is locked for the \_duration of the payment* - ie, until either the payment completes or times out.
+
+In an SCN, collateral to secure channels between A and B is locked along a similar route of intermediaries, and closing balance adjustments between A and B are pairwise adjusted along the route. Capital in intermediary channels is locked for the _duration of the virtual channel_.
+
+## Differences from Nitro-SCN
+
+### Ecosystem / Context
+
+The Bitcoin ecosystem has a strong intrinsic incentive toward participating in LN, as LN is the most (only?) viable solution to scaling BTC denominated payments with security anchored to bitcoin.
+
+Further: holders of BTC have relatively few options for activating their capital to produce a yield. Assets in EVM environments can generate yield via any number of defi protocols, including Ethereum proof-of-stake (staking pools offer approx %4 yield with near zero overhead).
+
+### Technical
+
+In the dominant use case, PCN payments are brief. The capital lockup time for an individual payment is `O(n)`, where `n` is length of the chain of intermediaries. LN strives toward a mesh network topology so that
```

Comment:
The PCN / SCN distinction works well for this document but I worry it could be a source of confusion. If we stick to payments in nitro protocol, we would technically have a Payment Channel Network, no?

Maybe we could use HTLC-PCN or something?

Reply:
PCN is pretty well established in "the literature" to refer to a LN style network. EG, a network of payment channels, which can multi-hop payments via HTLCs.

Even where nitro is used exclusively to open stateless voucher based payment channels, the base layer is made of state channels, making the PCN label unsuitable I think.